app/contexts/DeFiScanContext.tsx

```diff
@@ -39,14 +39,18 @@ function getTxURLByNetwork (network: EnvironmentNetwork, txid: string): string {
       break
 
     case EnvironmentNetwork.TestNet:
-      baseUrl = `${baseUrl}?network=${EnvironmentNetwork.TestNet}`
+      baseUrl += `?network=${EnvironmentNetwork.TestNet}`
       break
 
     case EnvironmentNetwork.LocalPlayground:
     case EnvironmentNetwork.RemotePlayground:
-      baseUrl = `${baseUrl}?network=${EnvironmentNetwork.RemotePlayground}`
+      baseUrl += `?network=${EnvironmentNetwork.RemotePlayground}`
       break
   }
 
+  if (typeof hexTxId === 'string' && hexTxId.length !== 0) {
+    baseUrl += `&rawtx=${hexTxId}`
```

Comment:
How about for main net handling where ? wasn't prefixed?

Reply:
Does mainnet uses a different rawtx value?What are the special

Reply:
MainNet does comes with the ?network=MainNet making your query string invalid

Reply:
Got what you mean now, sorry will update again